infrastructure/main.tf

```diff
@@ -85,13 +93,21 @@ module "wa_task_management_api_database_flexible_replica" {
     azurerm.postgres_network = azurerm.postgres_network
   }
 
-  source        = "git@github.com:hmcts/terraform-module-postgresql-flexible?ref=master"
-  product       = var.product
-  component     = var.component
-  name          = "${var.postgres_db_component_name}-postgres-db-flexible-replica"
-  location      = var.location
-  business_area = var.business_area
-  env           = var.env
+  source                      = "git@github.com:hmcts/terraform-module-postgresql-flexible?ref=master"
+  product                     = var.product
+  component                   = var.component
+  name                        = join("-", [local.db_name, "replica"])
+  location                    = var.location
+  business_area               = var.business_area
+  env                         = var.env
+  action_group_name           = join("-", [local.db_name, "replica", var.action_group_name])
+  cpu_threshold               = var.cpu_threshold
```

Comment:
provide cpu_threshold, memory_threshold and storage_threshold only when configurations is different than 80

otherwise it will use default values provided in terraform-module-postgresql-flexible